tools/bx.ps1

```diff
@@ -20,6 +20,11 @@ $targets = $Metaproj.Project.Target
 # For Conhost\Server, this will match:
 #   [Conhost\Server, Conhost\Server:Clean, Conhost\Server:Rebuild, Conhost\Server:Publish]
 $matchingTargets = $targets | Where-Object { $_.MSBuild.Condition -eq $msBuildCondition }
+if ($matchingTargets.length -eq 0)

```

Comment:
hold up, are you gonna conflict yourself? why does every PR have changes to bx ☹️

Reply:
AH that shouldn't be there!

The bcz change should be, but I was leaving the bx change in the PR it came from, which unfortunately, isn't merged, and I keep temporarily copying locally so I can build things

Reply:
(thanks)